css/css-multicol/column-count-used-001.html

```diff
@@ -1,19 +1,26 @@
 <!doctype html>
 <title>column-count used value when neither column-count nor column-width are auto</title>
 <link rel=help href=https://www.w3.org/TR/css-multicol-1/#pseudo-algorithm>
-<link rel=match href=column-count-used-001-ref.html>
+<link rel=match href=../reference/ref-filled-green-100px-square.xht>
 <style>
-ul {
-    font: 20px/1 Ahem;
-    width: 10em;
-    list-style-type: none;
-    columns: 2 2em;
+div {
+    position: absolute;
+}
+.bg {
+    background: red;
+    width: 100px;
+    height: 100px;
+}
+.test {
+    font: 50px/1 sans-serif;
```

Comment:
Use line-height:50px instead of specifying font, and there should be less trouble.